noir-projects/aztec-nr/aztec/src/oracle/shared_secret.nr

```diff
@@ -0,0 +1,16 @@
+use dep::protocol_types::{
+    address::aztec_address::AztecAddress,
+    point::{Point, POINT_LENGTH},
+    traits::Deserialize,
+};
+
+#[oracle(getSharedSecret)]
+pub unconstrained fn get_shared_secret_oracle(
+    address: AztecAddress,
+    ephPk: Point,
+) -> [Field; POINT_LENGTH] {}
```

Comment:
When I directly returned Point from the oracle, I got an error with num destination slots. I see that there is not other case that returns a multi-field value from oracle. Is it really just unsupported? I would expect Noir to manage to map the struct fields.

Reply:
Good point. Will investigate what did I do wrong here. Thanks